src/solver/hydro/management/management.cpp

```diff
@@ -102,17 +103,22 @@ void HydroManagement::prepareInflowsScaling(uint numSpace)
         auto& ptchro = *NumeroChroniquesTireesParPays[numSpace][z];
 
         auto& inflowsmatrix = area.hydro.series->storage;
+        auto& mingenmatrix = area.hydro.series->mingen;
```

Comment:
The current function is named **prepareInflowsScaling**, so it's supposed to make tasks about inflows only.
But there are also minimum hydro generation tasks done.
We could rename this function so that its body strictly adheres to what it does, but for reasons of readability and isolation, we should split it into (at least) 2 functions.
We could even split the min generation task into **minGenerationScaling** and **checkMinGeneration**, I think that it would be better.